C51-327: Add Open in Popup icon

deb1990 opened the pull request:
## Overview
As part of this PR a button has been added in the Activity Details panel, to open the activity in a popup.

## How it looks
![after](https://user-images.githubusercontent.com/5058867/48055661-1ff1ee80-e1d6-11e8-951a-680ce7508e2b.gif)

![2018-11-06 at 4 36 pm](https://user-images.githubusercontent.com/5058867/48060508-2be3ad80-e1e2-11e8-97ca-5bbc088152cb.png)

## Technical Details
`civicase__activity-panel__maximise` class is added for the button, and using css it has been rotated to get the desired look.

jamienovick commented:
Is there a danger or users clicking this or close by mistake with it so close to close? What about the other side?

swastikpareek commented on scss/components/_activity-panel.scss:
```diff
@@ -42,12 +42,17 @@
 }
 
+.civicase__activity-panel__maximise {
+  transform: rotate(45deg);
```
Nice trick 😉

swastikpareek commented:
> Is there a danger or users clicking this or close by mistake with it so close to close? What about the other side? Would need to test and see?

Good point. But we have a tooltip which says `View in Popup` which will eliminate the accidental click on close button. 
What are your thoughts @deb1990 ?
cc @jamienovick

deb1990 commented:
@jamienovick @JonScreat 
I have added a new ticket [C51-332](https://compucorp.atlassian.net/browse/C51-332) to identify and add the hover state for all icons.
Also I have added a bit more spacing between the icons to reduce the chance of accidental click. Please let me know if this looks good.
![2018-11-06 at 4 36 pm](https://user-images.githubusercontent.com/5058867/48060523-343be880-e1e2-11e8-987a-33a9e86e7143.png)

swastikpareek commented:
Looks good to me. Approved 🥂
